### ISSUE TITLE
not recognising href tag

### ISSUE THREAD

**Issue by srjshinde1** [iclems/iOS-htmltopdf#34]
Hello 

PDF converted perfectly fine but when it comes to clickable event on some text or part of pdf it is not working. 

There is on  section in pdf "click here" which opens url which is not working in PDF. 

Can you please guide if it is possible? 

Html Code :- <a href="$clickhere" target="_blank">Click here</a>

Thanks

**Comment by skela** [comment]
Yeah I too would love this feature to be implemented, especially for things like local html links, but yeah any kind of link would be fine.
